index.html

```diff
@@ -870,10 +912,10 @@ <h3 class="usa-collection__heading">Principal owner 4</h3>
         });
       });
 
-      // Clear sibling "I decline..." fields with other fields are checked
+      // Clear sibling "I decline..." fields with other fields are checked or text inputted
```

Comment:
Really old typo unrelated to this PR:

```suggestion
      // Clear sibling "I decline..." fields when other fields are checked or text inputted
```